apps/aecore/lib/aecore/naming/name_commitment.ex

```diff
@@ -17,7 +17,7 @@ defmodule Aecore.Naming.NameCommitment do
   @typedoc "Structure of the NameCommitment Transaction type"
   @type t :: %NameCommitment{
           hash: binary(),
-          owner: Keys.pubkey(),
+          owner: Identifier.t(),
```

Comment:
Same as above -> This will break the serialization:
https://github.com/aeternity/elixir-node/blob/c76f802d029419e90ab86d841948c1bc3d4349fa/apps/aecore/lib/aecore/naming/name_commitment.ex#L75